_beats/libbeat/scripts/Makefile

```diff
@@ -132,7 +132,8 @@ clean:: ## @build  Cleans up all files generated by the build steps
 	@rm -f docker-compose.yml.lock
 	@rm -f ${BEAT_NAME} ${BEAT_NAME}.test ${BEAT_NAME}.exe ${BEAT_NAME}.test.exe
 	@rm -f _meta/fields.generated.yml fields.yml
```

Comment:
what do you think about restoring the `rm -rf _beats`?

Reply:
sorry, meant this for the other makefile

Reply:
cancel that, that will make it more complicated to keep files we're overriding, like the readme